src/components/Images/Images.js

```diff
@@ -1,31 +1,81 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import * as Cloudinary from 'cloudinary-react'
+import cloudinary from 'cloudinary-core'
+import useRequired from '../../hooks/useRequired.js'
+import useInvalid from '../../hooks/useInvalid.js'
 
 // https://cloudinary.com/documentation/image_transformation_reference
 
 export const CLOUDINARY_CLOUD_NAME = 'getethos'
 
-const publicIdBase = `https://res.cloudinary.com/${CLOUDINARY_CLOUD_NAME}/image/upload/`
+const cld = new cloudinary.Cloudinary({
+  cloud_name: CLOUDINARY_CLOUD_NAME,
+})
+
+const BREAKPOINTS = [600, 900, 1200]
```

Comment:
Do these correspond to the Media queries? If not, should we store them in a `constants.js` file somewhere?

Reply:
They correspond to media queries.

Reply:
I dont actually use them anymore 😅good catch

Reply:
I don't actually use them, instead I redefine the same thing later 😅 good catch

Reply:
Drew I’m seeing this without access to my laptop ATM...is this the direction we want to go?